index.html

```diff
@@ -224,7 +224,15 @@ <h2><a name="documents">Reading material</a></h2>
       </tr>
       <tr>
         <td><a target="_blank" href="http://www.iana.org/domains">IANA Domain Name Services</a></td>
-        <td>IANA maintaines and operates several key aspects of the DNS</td>
+        <td>IANA maintains and operates several key aspects of the DNS</td>
+      </tr>
+      <tr>
+        <td><a target="_blank" href="https://jpmens.net/dns">Blogged DNS links listinformation listing</a></td>
+        <td>Compilation of blogged DNS links from Jan-Piet Mens</td>
+      </tr>
+      <tr>
+        <td><a target="_blank" href="http://mens.de/:/altdnsbook">Alternative DNS Servers (Free)</a></td>
+        <td>Book written by Jan-Piet Mens (link published with permission)</td>
```

Comment:
There's no need to say " (link published with permission)" :)

Reply:
Ok. Fixed! (Removed)